src/tuned_models.jl

```diff
@@ -160,6 +160,11 @@ function TunedModel(; model=nothing,
                               "If `tuning=Explicit()`, any model in the "*
                               "range will do. ")
     
+    if (acceleration isa CPUThreads && 
+        acceleration_resampling isa CPUProcesses)
+        acceleration = CPUProcesses()
+        acceleration_resampling = CPUThreads()
+    end
```

Comment:
Why are we mutating the hyper-parameters here and not in `clean!`  ?

Reply:
yeah. I'll fix that